apps/rush-lib/src/data/RushConfiguration.ts

```diff
@@ -210,7 +210,7 @@ export default class RushConfiguration {
 
       // Ignore things that aren't actual files
       const stat: fsx.Stats = fsx.statSync(resolvedFilename);
-      if (!stat.isFile() && !stat.isSymbolicLink) {
+      if (!stat.isFile() && !stat.isSymbolicLink()) {
```

Comment:
>isSymbolicLink [](start = 34, length = 14)

The documentation said that this method only works for lstat. [https://nodejs.org/api/fs.html#fs_class_fs_stats](https://nodejs.org/api/fs.html#fs_class_fs_stats)